sources/api/migration/migrator/tests/data/repository/targets/a-second-migration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Can we compress these as part of the testing rather than including binary artifacts?  It's weird to have the binaries, and it's likely someone will forget to change the binary if they change the script.

Reply:
Not without signing the TUF repo on the fly, which I plan to do once we do a release of the `RepositoryEditor` from @zmrow.

Reply:
> Not without signing the TUF repo on the fly, which I plan to do once we do a release of the `RepositoryEditor` from @zmrow.

Coming soon! :)

Reply:
Addressed by:

> webern force-pushed the webern:pr/signed-migrations branch from f481a2d to 4443441